PubNubSwiftConsole/Classes/UI/CollectionView/CollectionViewController.swift

```diff
@@ -382,7 +382,7 @@ public class CollectionViewController: ViewController, TextViewCollectionViewCel
     
     public override func viewDidLoad() {
         super.viewDidLoad()
-        let layout = CollectionViewFlowLayout()
```

Comment:
as nifty as this is, leave this all changes out of this pull request, we aren't supporting other layouts yet